recipes/soqt/meta.yaml

```diff
@@ -0,0 +1,48 @@
+package:
+    name: soqt
+    version: "1.6.0a"
+
+source:
+    fn: soqt.tar.gz
+    url: https://github.com/FreeCAD/pivy/releases/download/0.6.4/soqt_1.6.0a.tar.gz
+    sha256: a36664382e6a35873b7d0f65a1d302bdb3198d8bf8ae80e7f2db10c5f3bf895f
+
+build:
+    number: 0
+
+requirements:
+    build:
+        - {{ compiler("cxx") }}
+        - ninja
+        - cmake 3.10.*
+    host:
+        - qt 5.*
+        - coin3d
+    run:
+        - qt 5.*
+        - coin3d
+test:
+    requires:
+        - cmake
+
+    files:
+        - CMakeLists.txt  # [unix]
+
+    commands:
+        - cmake .  # [unix]
```

Comment:
This seems to indicate that there are no tests for windows

Reply:
Normally cmake tests the compiler and this is the problem for windows as test_env != build_env. But I found a workaround recently. Let's see if this works.